console-subscriber/src/builder.rs

```diff
@@ -23,6 +23,10 @@ pub struct Builder {
     /// the aggregator task.
     pub(super) event_buffer_capacity: usize,
 
+    /// Fraction of the event buffer needed before the aggregator task starts
+    /// to flush it more aggresively.
```

Comment:
can we describe more of how this fraction is represented here? is it relative to the `event_buffer_capacity`, or is it, say, out of 100%?

Reply:
also, i think there's a typo 
```suggestion
    /// Fraction of the event buffer needed before the aggregator task starts
    /// to flush it more aggressively.
```